ripser/ripser.py

```diff
@@ -37,6 +37,82 @@
 from pyRipser import doRipsFiltrationDMSparse as DRFDMSparse
 
 
+def get_greedy_perm_pc(X, N = -1, metric="euclidean"):
```

Comment:
Could you run [Black](https://black.readthedocs.io/en/stable/) over all the code you touched please?